src/app/(common)/chat/page.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+
+import { useEffect, useMemo, useState } from 'react';
+import { useRouter } from 'next/navigation';
+import { createChatRoom } from '@/src/apis/chat/chat';
+import ChatList from '@/src/components/chat/chatlist/ChatList';
+import { getUserRole, useAuthStore } from '@/src/stores';
+import type { Chat } from '@/src/types/chat';
+
+/**
+ * 임시 하드코딩: 모델(userId=3) ↔ 디자이너(userId=2) 1:1 방 1개만 리스트로 노출
+ */
+export default function ChatPage() {
+  const router = useRouter();
+  const storeRole = useAuthStore((state) => state.user?.role);
+  const [role, setRole] = useState<'model' | 'designer' | null>(null);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    const resolved = storeRole ?? getUserRole();
+    setRole(resolved);
+  }, [storeRole]);
+
+  const targetUserId = useMemo(() => {
+    if (role === 'designer') return 3; // 디자이너 -> 모델 userId=3
+    if (role === 'model') return 2; // 모델 -> 디자이너 userId=2
+    return null;
+  }, [role]);
+
+  const counterpartLabel = useMemo(() => {
+    if (role === 'designer') return '모델(userId=3)';
+    if (role === 'model') return '디자이너(userId=2)';
+    return '상대방 정보 없음';
+  }, [role]);
```

Comment:
중복 로직 객체 매핑 후 useMemo 방식으로 변경
```suggestion
const roleMappings = {
  designer: { targetUserId: 3, counterpartLabel: '모델(userId=3)' },
  model: { targetUserId: 2, counterpartLabel: '디자이너(userId=2)' }
} as const;

const { targetUserId, counterpartLabel } = useMemo(() => 
  role ? roleMappings[role] : { targetUserId: null, counterpartLabel: '상대방 정보 없음' },
  [role]
);
```